src/content/references/en/4-crophype.md

```diff
@@ -0,0 +1,21 @@
+---
+image: ../../../images/references/crophype.png
+title: EnMap and Sentinel 2 project portal for CropHype
+---
+
+We develop methods for integration and pipelines for processing data from DLR’s latest hyperspectral satellite: EnMap. Based on our Geo Engine, these are deployed in the cloud and used to develop ML models. In addition, there is a connection to Sentinel-2 data, which is used for joint processing with the EnMap data.
+
+## Highlights
+
+- Integration of hyperspectral data from the EnMap satellite
+- Automation with processing pipelines and Analysis Ready Data (ARD)
+- Hosting in the cloud
+- Unified access / data mesh for heterogeneous data (EnMap, Sentinel, DEM, field information).
+
+## Methodik
```

Comment:
should be Methodology

Reply:
[304b434](https://github.com/geo-engine/geoengine.de/pull/6/commits/304b4343dd4c0ea1ab9fdb71988815b9b1b5c145)